examples/rbd/secret.yaml

```diff
@@ -10,3 +10,6 @@ stringData:
   # specified in the storage class
   userID: <plaintext ID>
   userKey: <Ceph auth key corresponding to ID above>
+data:
+  # Key value is a base64 encoded encryption passphrase
+  encryptionPassphrase: dGVzdA==
```

Comment:
as the value is already encoded can we use stringData instead of data?

Reply:
using stringData will have plaintext passphrase instead of base64 encoded version